.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,36 @@
+---
+name: "Bug report \U0001F41B" 
+about: "If something isn't working as expected \U0001F914."
+title: ''
+labels: bug
+assignees: J535D165, qubixes
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug with the front-end is.
```

Comment:
Is this only for the front-end?

Reply:
Huh, the "with the frond-end" part shouldn't be there. Thanks for your attentiveness!